### PR TITLE
Amish

### PR DESCRIPTION
**Pull request by AmishOm**

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>gp</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
```
Comment (ShreyaTestYantra) [inline .project]: Okay Fixed

Comment (AmishOm) [reply .project]: > Okay Fixed1

**Review by ShreyaTestYantra — COMMENTED** [review]
Add Depencdeny in pom.xml

**Comment by AmishOm** [comment]
> Add Depencdeny in pom.xml

will be added

**Pull request merged**